include/neml2/tensors/user_tensors/Orientation.h

```diff
@@ -48,8 +48,10 @@ class Orientation : public Rot, public NEML2Object
 
 private:
   /// A helper method to dispatch to the correct fill method based on the selected options
-  Rot
-  fill(const std::vector<Real> & values, std::string input_type, const OptionSet & options) const;
+  Rot fill(const std::vector<Real> & values,
+           std::string input_type,
+           bool normalize,
```

Comment:
Consider adding a docstring for this `normalize` parameter. It wasn't clear to me what this option does before I read the source code.